magento/inventory
# Msi 1041

### Description (*)
Add admin grid mass actions for enabling and disabling stock sources

### Fixed Issues (if relevant)

1. magento-engcom/msi#1041: Cannot disable multiple Sources by one operation like with Stocks

### Manual testing scenarios (*)
1. Navigate to inventory/source/index
2. Mass action for enable / disable multiple sources should be available
3. default source stock will not be changed by the mass actions
